tsi_core/src/main.c

```diff
@@ -39,10 +36,10 @@ int main(void)
 	printf("                |||||   |||||\n");
 	printf("                  |||   |||\n");
 	printf("                        |\n");
-
-        LOG_INF("Test Platform: %s",CONFIG_BOARD_TARGET);
-        LOG_WRN("Testing on FPGA");
-	printk("TSI Logging enabled and printk is functional\n");
+  
+  LOG_INF("Test Platform: %s", CONFIG_BOARD_TARGET);
+	LOG_WRN("Testing on FPGA");
+	printk("TSI Logging enabled & printk is functional\n");
```

Comment:
We should have a config parameter that indicates logging enabled vs disabled, and check against that config variable to print this message. Right now, this is being printed statically.